src/index.css

```diff
@@ -23,7 +23,11 @@
   html,
   body,
   #root {
-    @apply text-white bg-black font-text w-full min-h-screen;
+    @apply text-white bg-black font-text min-h-screen;
+  }
+
+  html {
+    scrollbar-gutter: stable both-edges; /* Prevent layout shifting */
```

Comment:
It's not supported by Safari because Safari's scrollbar is overlayed by default. On chrome (at least on windows) the gutter changes the width of the viewport when it's displayed or not. Which leads to layout shifting.

Reply:
Got it - thanks!